server/upsd.c

```diff
@@ -1270,7 +1270,16 @@ int main(int argc, char **argv)
 	poll_reload();
 
 	if (num_ups == 0) {
-		fatalx(EXIT_FAILURE, "Fatal error: at least one UPS must be defined in ups.conf");
+		char *envvar = getenv("NUT_NOCONF_ALLOWED");
```

Comment:
Nitpicking, but I'd rather have `const char *envchar` instead of `char *envchar` since we won't modify the contents of `envchar`.

Reply:
Does it change much in practice, especially as we forget it when exiting the if-scope?

Reply:
Fixed in a later PR